### PR TITLE
fix for invalid cert path when host name is set in upper-case letters

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -1,5 +1,6 @@
 class foreman_scap_client::params {
+  $fqdn_s = downcase($fqdn)
   $ca_file          = pick($::rh_certificate_repo_ca_file, '/var/lib/puppet/ssl/certs/ca.pem')
-  $host_certificate = pick($::rh_certificate_consumer_host_cert, "/var/lib/puppet/ssl/certs/${fqdn}.pem")
-  $host_private_key = pick($::rh_certificate_consumer_host_key, "/var/lib/puppet/ssl/private_keys/${fqdn}.pem")
-}
+  $host_certificate = pick($::rh_certificate_consumer_host_cert, "/var/lib/puppet/ssl/certs/${fqdn_s}.pem")
+  $host_private_key = pick($::rh_certificate_consumer_host_key, "/var/lib/puppet/ssl/private_keys/${fqdn_s}.pem")
+}
```
Comment (shlomizadok) [inline manifests/params.pp]: Nitpick: EOF is missing.

File: manifests/params.pp [hunk 2]
```diff
@@ -1,5 +1,6 @@
 class foreman_scap_client::params {
+  $fqdn_s = downcase($fqdn)
```
Comment (shlomizadok) [inline manifests/params.pp]: Better variable name? `$downcased_fqdn` ?

**Comment by shlomizadok** [comment]
Not sure if this should be done here or downcase it on foreman_openscap.
@ares, thoughts?

**Comment by ares** [comment]
I think it's fine to do it here as well. It's unlikely to have file names in upper case. foreman_openscap could do the same when constructing the policy enc part, I don't think it's too bad to have it on both places.

**Comment by shlomizadok** [comment]
@allomani - Thanks for this important pull request.
I have two small nitpicks (Add extra line, change variable name) and I believe we're good to go.

**Comment by shlomizadok** [comment]
Perfect. Thanks @allomani !

**Pull request merged**